chain/bitcoind_client.go

```diff
@@ -245,12 +245,46 @@ func (c *BitcoindClient) NotifyReceived(addrs []btcutil.Address) error {
 func (c *BitcoindClient) NotifySpent(outPoints []*wire.OutPoint) error {
 	_ = c.NotifyBlocks()
 
+	// Send the outpoints so the client will cache them.
 	select {
 	case c.rescanUpdate <- outPoints:
 	case <-c.quit:
 		return ErrBitcoindClientShuttingDown
 	}
 
+	// Now we do a quick check in current mempool to see if we already have
+	// txes that spends the given outpoints.
+	for _, op := range outPoints {
```

Comment:
I thought the idea was to push this up a bit into lnd's existing chain notification system?

Reply:
I think we want to do it here as we already have the mempool data available.

Reply:
Yep makes sense.